### DISCUSSION TITLE
how to specify Index of GPU in textpredictor，does it support multiple CPUs working at the same time?

### DISCUSSION THREAD

**Discussion by bioinformatica** [autogluon/autogluon#2320] [Q&A] [answered]
My computer now has 4 gpus

**Comment by sxjscience** [comment, marked as answer]
The default behavior is that we will auto-select an empty GPU. What you can do is to follow https://stackoverflow.com/questions/39649102/how-do-i-select-which-gpu-to-run-a-job-on and set CUDA_VISIBLE_DEVICES flag.
